tests/manage/z_cluster/nodes/test_nodes_maintenance.py

```diff
@@ -506,6 +521,27 @@ def test_pdb_check_simultaneous_node_drains(
         if not sample.wait_for_func_status(result=False):
             log.error("Blocking PDBs still exist")
 
+        # Check Mon failovers in 10mins, 15mins and 20 mins
+        timeout = [600, 300, 300]
```

Comment:
Same as above.
The intervals mentioned in the comment are 10mins, 15mins and 20 mins. So, the timeout values here should be
timeout = [600, 900, 1200]

Reply:
Same as above

Reply:
Ok, the added comment is confusing, can you please elaborate and add a clear mention about it.